src/firefly/js/ui/VoSearchPanel.jsx

```diff
@@ -79,8 +85,27 @@ var sizeArea = () => {
     );
 };
 
-function voSearchArea() {
+var voSearchArea = () => {
     return (
-        <div>Search registry box, table result and cone search field here!</div>
+        <ValidationField
+            fieldKey={'vourl'}
+            /*validator={urlValidator}*/
+            tooltip='Enter the VO cone search URL directly (or use the link below to open external NVO search and find the VO cone search URL)'
```

Comment:
You are right, it doesn't. I will fix that. Need to be in a 'initialState' object to be set properly.